www/md_base/src/icons/gear-outline.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
not sure about the svg. I'd prefer to stick with a font like font-awesome, which is so much lighter